docs/internals/contributing/writing-documentation.txt

```diff
@@ -45,11 +45,16 @@ documentation is transformed into HTML, PDF, and any other output format.
 __ https://www.sphinx-doc.org/
 __ https://docutils.sourceforge.io/
 
-To build the documentation locally, install Sphinx:
+To build the documentation locally, install the doc requirements from the
```

Comment:
I think it should say "from the project root, install the documentation building requirements." The current ordering could be interpreted that the doc requirements are in the project root.